pkgs/applications/networking/browsers/tor-browser-bundle-bin/mutable.nix

```diff
@@ -0,0 +1,127 @@
+{ lib
+, buildFHSUserEnv
+, fetchurl
+, writers
+, tor-browser-bundle-bin
+
+# For XDG_DATA_DIRS
+, gnome3 # .adwaita-icon-theme
+, shared-mime-info
+, glib
+, gsettings-desktop-schemas
+, gtk3
+
+# override the bootstrapping script, useful for debugging
+, runScript ? null
+
+, audioSupport ? mediaSupport
+, pulseaudioSupport ? false
+# Media support (implies audio support)
+, mediaSupport ? true
+
+# Hardening
+, graphene-hardened-malloc
+# crashes with intel driver
+, useHardenedMalloc ? false
+
+}:
+
+let
+  lang = "en-US";
+  inherit (tor-browser-bundle-bin.meta.passthru) src srcs version;
+
+  # Bootstrapping script
+  tor-browser-bootstrap = writers.writeBash "tor-browser-bootstrap" ''
+    set -o errexit -o pipefail
+
+    if ! [[ -v XDG_DATA_HOME || -v HOME ]]
+    then
```

Comment:
```suggestion
    if ! [[ -v XDG_DATA_HOME || -v HOME ]]; then
```

Reply:
Is there a bash style guide or something like that for nixpkgs?

Reply:
I always follow shellcheck but not one in nix I know of but this is pretty standard to do with ;.

Reply:
makes sense to follow shellcheck I guess